### PR TITLE
Initial Version of the terraform-github-organization module

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+output "username" {
+  value = { for k, v in github_membership.membership : k => v.username }
+}
+
+output "role" {
+  value = { for k, v in github_membership.membership : k => v.role }
+}
```
Comment (mariux) [inline outputs.tf]: why not just export `github_membership.membership` as-is?

File: test/github_organization_test.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	t.Parallel()
+
+	terraformOptions := &terraform.Options{
+		// The path to where your Terraform code is located
+		TerraformDir: "../examples/organization",
+		Upgrade:      true,
+		// Variables to pass to our Terraform code using -var options
+		Vars: map[string]interface{}{
+			"github_token":        githubToken,
+			"github_organization": githubOrganization,
```
Comment (mariux) [inline test/github_organization_test.go]: is this needed as the github provider should pickup the environment anyway?

File: outputs.tf (hunk 1 above)
Comment (soerenmartius) [reply outputs.tf]: yeah good idea!

File: test/github_organization_test.go (hunk 2 above)
Comment (soerenmartius) [reply test/github_organization_test.go]: good point! It is not necessarily needed!

**Pull request merged**